pkg/capture/capture.go

```diff
@@ -23,15 +23,24 @@ const (
 
 	// ErrorThreshold is the maximum amount of consecutive errors that can occur on an interface before capturing is halted.
 	ErrorThreshold = 10000
+
+	// localBufferSizeLimit denotes the maximum buffer size available during rotation
+	localBufferSizeLimit = 8 * 1024 * 1024 // 8 MiB
```

Comment:
This is currently a constant. I have no good feeling on how much it will take on productive hosts to cover the "downtime" during rotation TBH. I also can't really estimate what the actual memory footprint will be on busy hosts (as it draws from a memory pool it's certainly not n_iface * max_buf_size, but there is _some_ concurrency while draining the buffer). We might also make this configurable of course, but I'd like to get a feeling first before propagating this all the way to the config / command line.